INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6042', 'GEUVADIS: Genetic European Variation in Disease', 'GEUVADIS: Genetic European Variation in Disease (http://www.geuvadis.org), is a European Medical Sequencing Consortium aiming at sharing capacity across Europe in high-throughput sequencing technology to explore genetic variation in health and disease. It is funded by the European Commission 7th framework program under the Coordination and Support Action scheme. It started on the 1st October 2010, and ends on 31st December 2013. The project Coordinator is Xavier Estivill from Center for Genomic Regulation, Barcelona.', 'Germline', 'Aggregate', 'Center for Genomic Regulation - CRG (Barcelona); Institute of Human Genetics - Helmholtz Zentrum (Munich)', 'Center for Genomic Regulation - CRG (Barcelona), Institute of Human Genetics - Helmholtz Zentrum (Munich)', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB629', 'Sequencing of five barley cultivars.', 'Hordeum vulgare subsp. vulgare', 'Germline', 'Control Set', NULL, 'IPK-Gatersleben', 112509);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB3025', 'Whole genome shotgun survey sequencing of Barley cv. Igri for de novo assembly of gene-space', 'Variation data is shown from four cultivar (Barke, Bowman, Igri, Haruna Nijo) and a wild barley (H. spontaneum).', 'Germline', 'Control Set', 'Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), D-06466 Seeland OT Gatersleben, Germany; Leibniz Institute of Age Research- Fritz Lipmann Institute (FLI), D-07745 Jena, Germany', 'IPK-Gatersleben', 112509);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB3042', 'Whole genome shotgun survey sequencing of Barley cv. Haruna Nijo for de novo assembly of gene-space', 'Variation data is shown from four cultivar (Barke, Bowman, Igri, Haruna Nijo) and a wild barley (H. spontaneum).', 'Germline', 'Control Set', 'Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), D-06466 Seeland OT Gatersleben, Germany; Leibniz Institute of Age Research- Fritz Lipmann Institute (FLI), D-07745 Jena, Germany', 'IPK-Gatersleben', 112509);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB5978', 'NextGen project variation for Capra aegagrus', 'Variation site discovery and genotype calls in the genus Capra (goat) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options).', 'Germline', 'Control Set', NULL, 'EBI', 9923);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6024', 'Sheep EVA Tax ID 9940 update', 'Variation site discovery and genotype calls in the genus Ovis (sheep) (Tax ID 9940) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options)', 'Germline', 'Control Set', NULL, 'EBI', 9940);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB3041', 'Whole genome shotgun survey sequencing of Barley cv. Barke for de novo assembly of gene-space', 'Variation data is shown from four cultivar (Barke, Bowman, Igri, Haruna Nijo) and a wild barley (H. spontaneum).', 'Germline', 'Control Set', 'Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), D-06466 Seeland OT Gatersleben, Germany; Leibniz Institute of Age Research- Fritz Lipmann Institute (FLI), D-07745 Jena, Germany', 'IPK-Gatersleben', 112509);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB3027', 'Whole genome shotgun survey sequencing  of Barley cv. Morex for de novo assembly of gene-space', 'Barley (Hordeum vulgare L.) is amongst the oldest domesticated crop plants and remains one of the world''s most important crop species. It is diploid with a haploid genome of 5.1 gigabases (Gb), twice the size of those of human and maize, and closely related to the most widely grown crop, hexaploid wheat. To meet global demand for food, feed and fibre, it is commonly agreed that reference genome sequences of our crop plants are urgently required to enable genome-assisted crop improvement. In the framework of the The International Barley Genome Sequencing Consortium (IBSC) we present here the whole genome shotgun rawdata of different barley cultivars and genotypes to build a physical ordered genetic and functional sequence resource.', 'Germline', 'Control Set', 'Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), D-06466 Seeland OT Gatersleben, Germany; Leibniz Institute of Age Research- Fritz Lipmann Institute (FLI), D-07745 Jena, Germany', 'IPK-Gatersleben', 112509);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB3040', 'Whole genome shotgun survey sequencing of Barley cv. Bowman for de novo assembly of gene-space', 'Variation data is shown from four cultivar (Barke, Bowman, Igri, Haruna Nijo) and a wild barley (H. spontaneum).', 'Germline', 'Control Set', 'Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), D-06466 Seeland OT Gatersleben, Germany; Leibniz Institute of Age Research- Fritz Lipmann Institute (FLI), D-07745 Jena, Germany', 'IPK-Gatersleben', 112509);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6911', 'Sanger Institute Mouse Genomes Project v3', 'SNP and INDEL calls from version 3 of the Mouse Genome Project at the Wellcome Trust Sanger Institute. Specifically, this project describes the variants of 18 mouse strains (129P2, 129S1, 129S5, A/J, AKR/J, BALB/cJ, C3H/HeJ, C57BL/6NJ, CAST/EiJ, CBA/J, DBA/2J, FVB/NJ, LPJ, NOD/ShiLtJ, NZOHILtJ, PWK/PhJ, SPRET/EiJ, WSB/EiJ) called against the reference mouse genome sequence GRCm38.', 'Germline', 'Control Set', NULL, 'Wellcome Trust Sanger Institute', 10088);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6506', 'NextGen project variation for Ovis vignei (sheep)', 'Variation site discovery and genotype calls in the genus Ovis vignei (sheep) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options)', 'Germline', 'Control Set', NULL, 'EBI', 59896);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7217', 'UK10K Avon Longitudinal Study of Parents and Children (ALSPAC) Variants', 'The UK10K project proposes a series of complementary genetic approaches to find new low-frequency/rare variants contributing to disease phenotypes. These will be based on obtaining the genome-wide sequence of 4000 samples from the Department of Twin Research and Genetic Epidemiology (TwinsUK) and Avon Longitudinal Study of Parents and Children (ALSPAC) cohorts (at 6x sequence coverage), and the exome sequence (protein-coding regions and related conserved sequence) of 6000 samples selected for extreme phenotypes. This project describes the variants identified from the ALSPAC study. For further information with regard to this cohort please contact Brent Richards (brent.richards@kcl.ac.uk) or Nicole Soranzo (ns6@sanger.ac.uk).', 'Germline', 'Control Set', 'Wellcome Trust Sanger Institute', 'SC', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7923', 'Vervet Genetic Mapping Project', 'The Vervet Genetic Mapping Project (VGMP) devloped genomic tools and resources for genetic trait maping in vervet monkey (AKA African green monkey). This project generated genome-wide SNP panels dedicated specifically for genetic mapping in Caribbean-origin sabaeus vervets (Chlorocebus aethiops sabaeus). The Assocation Mapping Panel comprises of ~500K SNPs (r2=0.9). The Linkage Mapping Panel comprises of ~150K SNPs (r2=0.5).', 'Germline', 'Control Set', NULL, 'University of California Los Angeles', 60711);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB8636', 'deCODE Genetics Whole-Genome Sequencing Variants', 'Autosomal sequence variants identified in deCODE''s whole-genome sequencing of 2,636 Icelanders', 'Germline', 'Control Set', NULL, 'deCODE Genetics', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB9373', 'A multipurpose high throughput SNP chip for the dengue and yellow fever mosquito, Aedes aegypti', 'Genotyping of multiple Aedes aegypti mosquito populations worldwide using a novel SNP chip.', 'Germline', 'Control Set', NULL, 'Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06511, USA', 7159);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB8639', 'GenomeDK Release 1 SoapAsmVar-Indels calls', 'This is an aggregate vcf of SoapAsmVar Indels calls (10nt-50nt) from 10 trios sequenced to high depth from the first phase Danish Pangenome project', 'Germline', 'Control Set', NULL, 'Technical University of Denmark; Aarhus University; Copenhagen University; BGI-Europe', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB9507', 'Whole-genome Sequencing Reveals Untapped Genetic Potential in Sorghum', 'These data report 8â€‰M high-quality SNPs, 1.9â€‰M indels and specific gene loss and gain events in S. bicolor.', 'Germline', 'Control Set', NULL, 'Fisheries and Forestry Queensland', 4557);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB9857', 'Prognostic factors of early response to TKI in chronic myeloid leukemia by whole exome sequencing', 'Chronic myeloid leukemia can be well controlled by targeted tyrosine kinase inhibitors (TKI). However there is about 10% of patients with primary resistance or low efficacy of the therapy. In our study we found promissing new potential early prognostic markers and molecular targets for developing new drugs. Â ', 'Germline', 'Case Control', NULL, 'Federal State Budgetary Institution Â«Research Centre For Medical GeneticsÂ», Moscow, Russia', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB8650', 'GenomeDK Release 1 SNV calls', 'This is an aggregate vcf of GATK SNV calls from 10 trios sequenced to high depth from the first phase Danish Pangenome project', 'Germline', 'Control Set', NULL, 'Technical University of Denmark; Aarhus University; Copenhagen University; BGI-Europe', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB8652', 'GenomeDK Release 1 GATK-Indels calls', 'This is an aggregate vcf of GATK Indels calls from 10 trios sequenced to high depth from the first phase Danish Pangenome project', 'Germline', 'Control Set', NULL, 'Technical University of Denmark; Aarhus University; Copenhagen University; BGI-Europe', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB8661', 'The Exome Aggregation Consortium (ExAC) v0.3', 'The Exome Aggregation Consortium (ExAC) is a coalition of investigators seeking to aggregate and harmonize exome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community.', 'Germline', 'Control Set', NULL, 'The Eli and Edythe L. Broad Institute of Harvard and MIT', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB9858', 'Prognostic factors of stable remission after cessation of TKI therapy in chronic myeloid leukemia by whole exome sequencing', 'Chronic myeloid leukemia can be well controlled by targeted tyrosine kinase inhibitors (TKI). In some patients it is possible to terminate expensive and toxic long-life therapy, however about 50% of these patients relapse without treatment. In our study we found promissing new potential early prognostic markers and molecular targets for developing new drugs.', 'Germline', 'Case Control', NULL, 'Federal State Budgetary Institution Â«Research Centre For Medical GeneticsÂ», Moscow, Russia', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7378', 'NextGen project variations for Ovis (sheep)', 'Variation site discovery and genotype calls in the genus Ovis (sheep) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options).', 'Germline', 'Control Set', NULL, 'EBI', 9935);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB9656', 'Prognostic factors of early response to TKI and remission stability in chronic myeloid leukemia by whole exome sequencing', 'Chronic myeloid leukemia can be well controlled by targeted tyrosine kinase inhibitors (TKI). In some patients it is possible to terminate expensive and toxic long-life therapy, however about 50% of these patients relapse without treatment. There are also about 10% of patients with primary resistance or low efficacy of the therapy. In our study we found promising new potential early prognostic markers and molecular targets for developing new drugs.', 'Germline', 'Case Control', NULL, 'Federal State Budgetary Institution Â«Research Centre For Medical GeneticsÂ», Moscow, Russia', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB8705', 'MGP: Medical Genome Project healthy controls from Spanish population', 'MGP contains aggregated information on 267 healthy individuals, representative of the Spanish population that were used as controls in the MGP (Medical Genome Project; http://www.medicalgenomeproject.com)', 'Germline', 'Control Set', NULL, 'GBPA (http://www.gbpa.es/); CIPF (http://bioinfo.cipf.es/); IBIS (http://www.ibis-sevilla.es/)', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6930', '1000 Genomes Project phase3 release V3+', 'This submission contains variant sites and phased genotypes for all autosomes (excluding a 3Mbps region on chromosome 12 between 7M and 10M) derived from the 1000 Genomes Project phase3 sequence data. ~3400 unphased sites were removed from the submission while they are being investigated. The missing data on chromosome 12 are being patched up and will be submitted at a later time together with the sex chromosomes. This data set is based on 20130502 sequence freeze and alignments; which were collected from 2504 unrelated individuals from 26 populations. The analysis was run using only Illumina platform sequence and only considered sequence with read lengths of 70bp or greater.', 'Germline', 'Control Set', NULL, 'EBI', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6040', 'UK10K', 'The UK10K project proposes a series of complementary genetic approaches to find new low-frequency/rare variants contributing to disease phenotypes. These will be based on obtaining the genome-wide sequence of 4000 samples from the TwinsUK and ALSPAC cohorts (at 6x sequence coverage), and the exome sequence (protein-coding regions and related conserved sequence) of 6000 samples selected for extreme phenotypes. Our studies will focus primarily on cardiovascular-related quantitative traits, obesity and related metabolic traits, neurodevelopmental disorders and a limited number of extreme clinical phenotypes that will provide proof-of-concept for future familial trait sequencing. We will directly analyse quantitative traits in the cohorts and the selected traits in the extreme samples, and also use imputation down to 0.1% allele frequency to extend the analyses to further sample sets with genome wide genotype data. In each case we will investigate indels and larger structural variants as well as SNPs, and use statistical methods that combine rare variants in a locus or pathway as well as single-variant approaches. The TwinsUK samples will be part of the cohort study and will undergo whole genome sequencing. For further information with regard to this cohort please contact Brent Richards (brent.richards@kcl.ac.uk) or Nicole Soranzo (ns6@sanger.ac.uk).', 'Germline', 'Control Set', 'Wellcome Trust Sanger Institute', 'SC', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6902', 'Complete Genomics 69 Genomes', 'A list of all of the variants in the set of 69 Complete Genomics genomes, generated using the CGA Tools (v1.5) listvariants command', 'Germline', 'Control Set', NULL, 'Complete Genomics', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB5966', 'Sheep EVA Tax ID 9940', 'Variation site discovery and genotype calls in the genus Ovis (sheep) (Tax ID 9940) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options)', 'Germline', 'Control Set', NULL, 'EBI', 9940);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6025', 'NextGen project variation for Ovis aries', 'Variation site discovery and genotype calls in the genus Ovis (sheep) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options)', 'Germline', 'Control Set', NULL, 'EBI', 9940);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB5829', 'Genome of the Netherlands Release 5', 'GoNL: Genome of the Netherlands Release 5', 'Germline', 'Control Set', 'The University Medical Center Groningen; Leiden University Medical Center; Erasmus Medical Center; Vrije Universiteit Amsterdam; University Medical Centre Utrecht', 'The University Medical Center Groningen, Leiden University Medical Center, Erasmus Medical Center, Vrije Universiteit Amsterdam, University Medical Centre Utrecht', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB5967', 'Sheep EVA Tax ID 9938', 'Variation site discovery and genotype calls in the genus Ovis (sheep) (Tax ID 9938) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options)', 'Germline', 'Control Set', NULL, 'EBI', 9938);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB4395', 'Whole genome re-sequencing in tomato', 'Whole genome re-sequencing in tomato reveals variation associated with introgression and breeding events. Here we describe the whole-genome resequencing of eight tomato samples chosen to represent a large range of intra-specific variability and the identification and annotation of novel polymorphisms.Comparison of the sequences with the reference genome yielded more than 4 million single nucleotide polymorphisms (SNPs) and almost 128,000 InDels were detected.This study represents the first whole genome resequencing experiment in cultivated tomato.', 'Germline', 'Control Set', NULL, 'Institut National de la Recherche Agronomique - France', 4081);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB4019', '1000 Genomes Phase 1 Analysis', 'Variant discovery and individual genotypes for 1092 individuals from the 1000 Genomes Project Phase 1 data release. This accession is a snapshot, as of June 26, 2013, of directory ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/integrated_call_sets Individual level genotype information is contained in a separate .vcf file for each chromosome; the variant site information (first eight columns of .vcf format) is also collected in a single sites-only .vcf file. The .vcf version 4.1 file format is described at http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41 Variant discovery and genotyping are based on both low coverage whole genome sequencing and deep coverage exon capture sequencing by the 1000 Genomes Project. The Phase 1 low coverage sequencing was done between 2008 and 2010 using Illumina (86.4%), AB SOLiD (13%) and Roche LS 454 (0.6%) sequencing technologies, with a raw sequencing data freeze date of Nov 23, 2010. Exon capture sequencing was done in 2010 and early 2011, using either the NimbleGen SeqCap_EZ_Exome_v2 or Agilent SureSelect_All_Exon_V2 exome capture reagents, with a freeze date of May 21, 2011. After filtering, variant site discovery yields a total of 38 M SNPs, 1.4 M short indels and 14 K large deletions (longer than 50 bp). The short indels and large deletions in particular have been stringently filtered in order to achieve the project goal of not more than 5% false discovery rate. All coordinates are relative to the GRCh37 human genome reference sequence. Site discovery and genotyping for SNPs are based on both low coverage and exome capture sequence data; for short indels, only low coverage data, and for large deletions, only the 946 individuals with Illumina low coverage sequence data are used. As a final step, statistical genotype imputation using the linkage disequilibrium between nearby sites refines the individual genotypes and merges information from all three variant types. Details of data generation and processing for this data set are given in supplementary material to the 1000 Genomes Phase 1 paper: 1000 Genomes Project Consortium, Goncalo R. Abecasis, Adam Auton, Lisa D. Brooks, Mark A. DePristo, Richard M. Durbin, Robert E. Handsaker, Hyun M. Kang, Gabor T. Marth, Gil A. McVean (2012). An integrated map of genetic variation for 1,092 human genomes. Nature, v.491, n.7422, pp.56-65, Nov 1, 2012, PMID: 23128226, doi: 10.1038/nature11632. Further details and supporting data are in other subdirectories under: ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1', 'Germline', 'Control Set', 'The 1000 Genomes Project Consortium', '1000 Genomes Project', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6033', 'NextGen project variation for Ovis (sheep) (Tax ID 9938)', 'Variation site discovery and genotype calls in the genus Ovis (sheep) (Tax ID 9938) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options)', 'Germline', 'Control Set', NULL, 'EBI', 9938);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6119', 'NextGen project variation for Bos taurus', 'Variation site discovery and genotype calls in the genus Bos (cattle) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options).', 'Germline', 'Control Set', NULL, 'EBI', 9913);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7218', 'UK10K The Department of Twin Research and Genetic Epidemiology (TwinsUK) Variants', 'The UK10K project proposes a series of complementary genetic approaches to find new low-frequency/rare variants contributing to disease phenotypes. These will be based on obtaining the genome-wide sequence of 4000 samples from the Department of Twin Research and Genetic Epidemiology (TwinsUK) and Avon Longitudinal Study of Parents and Children (ALSPAC) cohorts (at 6x sequence coverage), and the exome sequence (protein-coding regions and related conserved sequence) of 6000 samples selected for extreme phenotypes. This project describes the variants identified from the TwinsUK study. For further information with regard to this cohort please contact Brent Richards (brent.richards@kcl.ac.uk) or Nicole Soranzo (ns6@sanger.ac.uk).', 'Germline', 'Control Set', 'Wellcome Trust Sanger Institute', 'SC', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7061', 'NextGen project variation for Bos taurus x Bos indicus', 'Variation site discovery and genotype calls in the genus Bos (cattle) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options).', 'Germline', 'Control Set', NULL, 'EBI', 30523);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6057', 'NextGen project variation for Capra hircus', 'Variation site discovery and genotype calls in the genus Capra (goat) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options).', 'Germline', 'Control Set', NULL, 'EBI', 9925);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7723', 'Biology of Rare Alleles in Maize', 'This project improves our understanding of the genetic architecture of complex traits in maize and its wild relative, teosinte. Maize has a combination of life history, economic and societal value, and genetic tools that make it uniquely suited to studying genetic architecture. Additionally, the ability of molecular and population genetic polymorphism descriptors to aid identifcation of functional rare alleles is also a key outcome of this project as is the prediction of phenotype from genotype in diverse maize cultivars.', 'Germline', 'Control Set', NULL, 'Cornell University', 4577);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7894', 'Genetic divergence between Anopheles gambiae M and S forms', 'Genotyping of Anopheles gambia M and S form mosquitoes from African populations collected at Yaounde (Cameroon), Dodow (Ghana), Tororo (Uganda) and Antula (Guinea-Bissau).', 'Germline', 'Control Set', NULL, 'Liverpool School of Tropical Medicine; Address: Pembroke Place, Liverpool, L35QA', 7165);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6495', 'NextGen project variation for Ovis orientalis', 'Variation site discovery and genotype calls in the genus Ovis (sheep) by the NEXTGEN project (Next generation methods to preserve farm animal diversity by optimizing present and future breeding options)', 'Germline', 'Control Set', NULL, 'EBI', 469796);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7895', 'The National FINRISK Study', 'The FINRISK cohorts comprise the respondents of representative, cross-sectional population surveys that are carried out every 5 years since 1972, to assess the risk factors of chronic diseases (e.g. CVD, diabetes, obesity, cancer) and health behavior in the working age population, in 3-5 large study areas of Finland. DNA samples were collected in the following survey years: 1987, 1992, 1997, 2002, 2007, and 2012.', 'Germline', 'Aggregate', NULL, 'Institute for Molecular Medicine Finland', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB5439', 'Exome Variant Server NHLBI Exome Sequencing Project', 'The goal of the NHLBI GO Exome Sequencing Project (ESP) is to discover novel genes and mechanisms contributing to heart, lung and blood disorders by pioneering the application of next-generation sequencing of the protein coding regions of the human genome across diverse, richly-phenotyped populations and to share these datasets and findings with the scientific community to extend and enrich the diagnosis, management and treatment of heart, lung and blood disorders.', 'Germline', 'Aggregate', 'University of Michigan; The Broad Institute', 'University of Michigan and the Broad Institute', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB9822', 'Genetic evidence for an origin of the Armenians from Bronze Age mixing of multiple populations', 'Population genetics study of 138 samples looking for genetic evidence of an origin of the Armenians from Bronze Age mixing of multiple populations', 'Germline', 'Control Set', NULL, 'Wellcome Trust Sanger Institute', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB10956', 'Intra-tumor Genetic Heterogeneity in Rectal Cancer', 'Targetted confirmatory sequencing of heterogenous variants detected by exome sequencing of spatially disparate areas from six rectal tumors', 'Germline', 'Case Set', NULL, 'University of Michigan Medical School', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB12769', 'Diversity of Taiwan rice varieties', 'To study breeding and domestication trace of Taiwan', 'Germline', 'Control Set', NULL, 'IPMB, Academia Sinica', 4530);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB10964', 'Whole genome sequencing of elite rice cultivars as a comprehensive information resource for marker assisted selection', 'Whole genome sequencing and bioinformatic analysis of 104 rice cultivars including 54 elite lines from North America and Latin America', 'Germline', 'Control Set', NULL, 'International Center for Tropical Agriculture (CIAT)', 4530);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB12771', 'One TRIM line, two diversed phenotype', 'There were two diversed phenotype in one T-DNA insertional line. The project is about to investigate which gene or any factor cause such diversity via genomic tools.', 'Germline', 'Control Set', NULL, 'IPMB, Academia Sinica', 39947);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJX00001', 'Human Variation Data From dbSNP build 144', 'This project contains the human variant data from dbSNP build 144.', 'Germline', 'Aggregate', NULL, 'National Center for Biotechnology Information (NCBI)', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB6041', 'UMCG Cardio GenePanel screening', 'Genetic variants called in individuals screened for heritable cardiomyopathies using the UMCG Cardio GenePanels', 'Germline', 'Case Set', NULL, 'University of Groningen, University Medical Center Groningen, Department of Genetics, Groningen, the Netherlands', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB12770', 'Study the somatic variant due to tissue culture and transformation', 'For inspecting the level of the somatic variation from tissue culture and transformation, there were eight treatments applied on rice. The variety is Tainung67 (TNG67), which widely used in genomic study in Taiwan and with a large poplation T-DNA insertional lines, the Taiwan Rice Insertional Mutagenesis (TRIM).', 'Germline', 'Control Set', NULL, 'IPMB, Academia Sinica', 4530);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB11471', 'Sanger Institute Mouse Genomes Project v5', 'SNP and INDEL calls from version 5 of the Mouse Genome Project at the Wellcome Trust Sanger Institute. Specifically, this project describes the variants of 36 mouse strains aligned against the reference mouse genome sequence GRCm38.', 'Germline', 'Control Set', NULL, 'Wellcome Trust Sanger Institute', 10088);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13086', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles funestus samples.', 'Anopheles funestus subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 62324);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB7898', 'The Exome Aggregation Consortium (ExAC) v0.2', 'The Exome Aggregation Consortium (ExAC) is a coalition of investigators seeking to aggregate and harmonize exome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community.', 'Germline', 'Aggregate', NULL, 'The Eli and Edythe L. Broad Institute of Harvard and MIT', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13625', 'Whole genome resequencing of the human parasite Schistosoma mansoni reveals population history and effects of selection', 'Whole genome sequences of S. mansoni and S. rodhaini aligned to S. masoni v5.2 reference. Samples collected from Africa and Carribean to faciliate coalescence modelling of population history', 'Germline', 'Control Set', NULL, 'Wellcome Trust Sanger Institute', 6183);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13080', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles stephensi samples.', 'Anopheles stephensi subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 30069);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB14685', 'High quality variant calls from the Sheep genomes project - Run1', 'High quality variant discovery using multiple sheep breeds - Run1: Fastq files were aligned to Ovis aries reference genome version 3.1 using bwa mem. Duplicates were removed using Samtools rmdup. Local realignment around indels was performed using GATK tools RealignerTargetCreator and IndelRealigner. Variant calling on resulting BAM files was conducted independently using both Samtools mpileup and GATK Unified Genotyper. Filters were applied to each of the Samtools and GATK produced VCF files to remove lower quality variants. These filters included remove variants with 2 or more alleles, remove variants never observed on forward or reverse strands, remove variants with overall quality less than phred score 20, remove variants with mapping quality less than phred score 30, remove variants with minimum depth less than 10 across all samples, remove variants with the same base pair position, remove lower quality indel when indels closer than 10 base pairs, remove lower quality variant where variants closer than 3 base pairs, remove SNP within 5bp of an indel. GATK CombineVariants was used to produce a union set of the filtered variants. An intersect set containing those variants concordant between Samtools and GATK predictions was extracted from this union set using GATK SelectVariants to produce the final VCF files. (Software: bwa-0.7.12,GATK-3.4-46,samtools-1.2+htslib-1.2.1)', 'Germline', 'Control Set', NULL, 'ISGC', 9940);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB14878', 'Recombination in cattle', 'Sex differences in global recombination and genetics of recombination in cattle', 'Germline', 'Control Set', NULL, 'Unit of Animal Genomics, University of Liege, Belgium', 9913);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB11984', 'Somatic Mutations in QIMR Berghofer Melanoma Cell Line Collection Identified using Exome sequencing', 'Somatic mutations were identified in the QIMR Berghofer melanoma cell line collection using exome sequencing and comparing with matched germline', 'Germline', 'Case Control', NULL, 'QIMR', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB14879', 'Recombination in cattle - imputed', 'Sex differences in global recombination and genetics of recombination in cattle - imputed variants', 'Germline', 'Control Set', NULL, 'Unit of Animal Genomics, University of Liege, Belgium', 9913);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13083', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles minimus samples.', 'Anopheles minimus subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 112268);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13082', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles quadriannulatus samples.', 'Anopheles quadriannulatus subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 34691);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13084', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles merus samples.', 'Anopheles merus subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 30066);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13517', 'Transcriptome characterization and gene expression analyses of salt-tolerant and salt-sensitive varieties of Pokkali Oryza sativa following exposure to salinity stress', 'Sequence reads from Pokkali, a salt-sensitive variety, were aligned to the reference Japonica genome to the identify single nucleotide polymorphisms', 'Germline', 'Control Set', NULL, 'Oregon State University', 4530);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13085', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles melas samples.', 'Anopheles melas subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms. This subsection of the project contains data for Anopheles arabiensis.', 'Germline', 'Control Set', NULL, 'Broad Institute', 34690);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13087', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles farauti samples.', 'Anopheles farauti subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 69004);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13088', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles epiroticus samples.', 'Anopheles epiroticus subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 199890);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13090', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles arabiensis samples.', 'Anopheles arabiensis subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 7173);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB14684', 'Raw variant calls from the Sheep genomes project - Run1', 'Raw variant discovery using multiple sheep breeds - Run1: Fastq files were aligned to Ovis aries reference genome version 3.1 using bwa mem. Duplicates were removed using Samtools rmdup. Local realignment around indels was performed using GATK tools RealignerTargetCreator and IndelRealigner. Variant calling on resulting BAM files was conducted independently using both Samtools mpileup and GATK Unified Genotyper. A union set of the Samtools and GATK called variants was produced using GATK CombineVariants to produce the final VCF files. (Software: bwa-0.7.12,GATK-3.4-46,samtools-1.2+htslib-1.2.1)', 'Germline', 'Control Set', NULL, 'ISGC', 9940);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB14713', 'Whole-genome sequencing expands the mutational spectrum of autism with novel genes, causative CNVs and chromosomal rearrangements', 'Identification of genomic variants from whole-genome sequencing dataset of 32 Chinese trios with autism, including de novo mutations (DNMs), inherited variants, copy number variants (CNVs) and genomic structural variants (SVs).', 'Germline', 'Control Set', NULL, 'Wenzhou Medical University', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13089', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles culicifacies samples.', 'Anopheles culicifacies subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 139723);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13516', 'Transcriptome characterization and gene expression analyses of salt-tolerant and salt-sensitive varieties of IR29 Oryza sativa following exposure to salinity stress', 'Sequence reads from IR29, a salt-sensitive variety, were aligned to the reference Japonica genome to the identify single nucleotide polymorphisms', 'Germline', 'Control Set', NULL, 'Oregon State University', 4530);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13081', 'Highly evolvable malaria vectors: The genomes of 16 Anopheles mosquitoes. Anopheles sinensis samples.', 'Anopheles sinensis subset of project sequencing 16 anopheline vector genomes to complement and facilitate comparative analysis with the three other sequenced anophelines, Anopheles gambiae PEST, M and S forms.', 'Germline', 'Control Set', NULL, 'Broad Institute', 74873);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB14385', 'Genomic diversity and introgression in O. sativa reveal the impact of domestication and breeding on the rice genome', 'In this study, we utilize a novel 1,536 SNP panel genotyped across 395 diverse accessions ofÂ O. sativaÂ to study genome-wide patterns of polymorphism, to characterize population structure, and to infer the introgression history of domesticated AsianÂ rice.Â ', 'Germline', 'Control Set', NULL, 'Cornell University', 4530);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB2733', 'Schizosaccharomyces pombe genetic diversity', 'S.pombe is the "other yeast" genetic model system.  All 107 known wild isolates have been collected  around the world.  14 have already been sequenced.  The aim of the study is to sequence 96 samples (three overlapping with the previously sequenced 14) to generate a genetic diversity resource for S.pombe comparable to that available for  S.cerevisiae. This data is part of a pre-publication release. For information on the proper use of pre-publication data shared by the Wellcome Trust Sanger Institute (including details of any publication moratoria), please see http://www.sanger.ac.uk/datasharing/', 'Germline', 'Control Set', NULL, 'SC', 4896);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB14378', 'Genomewide SNP variation reveals relationships among landraces and modern varieties of rice', 'We initiated the OryzaSNP project (www.OryzaSNP.org) to discover genetic variation within 20 rice varieties and landraces. Most varieties belong to the 2 main groups, indica and japonica, including tropical and temperate japonica, whereas others represent the aus, deep water, and aromatic rice groups. Here, we describe the discovery of high-quality, nonredundant SNPs distributed across the entire genomes of the OryzaSNPset.', 'Germline', 'Control Set', NULL, 'International Rice Research Institute', 4530);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB14018', 'Whole Genome Sequencing of eight EAC cell linesâ€”ESO26, ESO51, FLO-1, JH-EsoAd1, OACM5.1 C, OACP4 C, OE33, SK-GT-4â€”and one esophageal high grade dysplasia cell line, CP-D.', 'Esophageal Adenocarcinoma (EAC) is highly mutated and molecularly heterogeneous. The number of cell lines available for study is limited and their genome has been only partially characterized. The availability of an accurate annotation of their mutational landscape is crucial for accurate experimental design and correct interpretation of genotype-phenotype findings. We performed high coverage, paired end whole genome sequencing on eight EAC cell linesâ€”ESO26, ESO51, FLO-1, JH-EsoAd1, OACM5.1 C, OACP4 C, OE33, SK-GT-4â€”all verified against original patient material, and one esophageal high grade dysplasia cell line, CP-D. We have made available the aligned sequence data and report single nucleotide variants (SNVs), small insertions and deletions (indels), and copy number alterations, identified by comparison with the human reference genome and known single nucleotide polymorphisms (SNPs). We compare these putative mutations to mutations found in primary tissue EAC samples, to inform the use of these cell lines as a model of EAC.', 'Germline', 'Case Control', NULL, 'CRUK CAMBRIDGE RESEARCH INSTITUTE', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB4163', 'Strongyloides ratti genetic diversity', 'The genome sequences of Strongyloides ratti, S. stercoralis, S. papillosus and S. venezuelensis are providing a clear view of genome diversity within this genus. These species include one human parasite, one parasite of veterinary relevance and two species from rodents, used as laboratory models. We now want to better understand this within-genus diversity, by also looking at the genome diversity with one species, specifically S. ratti. There are different isogenic lines of S. ratti available. (These other lines are recently wild, whereas the S. ratti line sequenced so far has a very long history of laboratory adaptation.) Further, phenotypic data from these lines show that they differ in important infection characteristics.  These isofemale lines of S. ratti differ in their fecundity (which, for a parasite is key to transmission potential), particularly in the presence of the host immune response (which, in general suppresses nematode reproduction). These line also differ in the extent to which they reproduce by sexual reproduction outside of the host (compared with the alternative, asexual reproduction). Analogously, this trait (sex vs. no sex) is also affected by the host immune response. Together, this means that these S. ratti lines differ in how they respond to the host immune response, which emphasies that this parasite''s interaction with their host is dynamic both on short (i.e. within a worms lifetime) and long (i.e. evolutionary) term time scales. Exploring the genome diversity within this species will therefore begin to illuminate our understanding of these phenotypic differences.', 'Germline', 'Control Set', NULL, 'SC', 34506);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB13618', 'The 3,000 rice genomes project', 'The 3000 Rice Genome Project is an international effort to sequence the genomes of 3,024 rice varieties from 89 countries. The project contains the variant calls of 3000 samples against the Nipponbare reference genome sequence. This data is available for anyone to use under the terms of the Toronto Statement.', 'Germline', 'Control Set', NULL, 'International Rice Research Institute (IRRI)', 4530);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJNA289433', 'Study of Major Depression in Chinese women', 'Study of Major Depression in Chinese women', 'Germline', 'Control Set', NULL, 'BGI-Shenzhen', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJNA273563', '1001 Genomes: A Catalog of Arabidopsis thaliana Genetic Variation', '1001 Genomes: A Catalog of Arabidopsis thaliana Genetic Variation', 'Germline', 'Control Set', NULL, 'The 1001 Genomes Project Consortium', 3702);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB19524', 'Samsung Medical Centre: 36 familal exomes', '36 familial exomes from the Samsung Medical Centre', 'Germline', 'Control Set', NULL, 'SGI', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB15385', 'Illumina Platinum Genomes calls for NA12877 and NA12878 against GRCh37', 'Illumina Platinum Genomes calls for NA12877 and NA12878 against GRCh37', 'Germline', 'Control Set', NULL, 'Illumina', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB15384', 'Illumina Platinum Genomes calls for NA12877 and NA12878 against GRCh38', 'Illumina Platinum Genomes calls for NA12877 and NA12878 against GRCh38', 'Germline', 'Control Set', NULL, 'Illumina', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB20009', 'Cattle genetic variation data from 16 animals across 28 samples', 'Cattle genetic variation data from 16 animals across 28 samples', 'Germline', 'Control Set', NULL, 'Iowa State University', 9913);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB15197', 'deCODE Genetics whole-genome sequencing variants 2', 'Sequence variants identified in deCODE''s whole-genome sequencing of 15,220 Icelanders', 'Germline', 'Control Set', NULL, 'deCODE Genetics', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB18671', 'Sequencing of common bean genotypes provides comprehensive resources for genetic studies and molecular breeding', 'Whole genome sequencing and bioinformatic analysis of 29 cultivars of common bean (Phaseolus vulgaris), nine pools and 2 cultivated relatives of P. coccineus and P. acutifolius', 'Germline', 'Control Set', NULL, 'CIAT, UCDavis, Universidad de los Andes', 3885);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB19523', 'Samsung Medical Centre: 85 panel samples', '85 panel samples from the Samsung Medical Centre', 'Germline', 'Control Set', NULL, 'SGI', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB15198', 'deCODE Genetics whole-genome de novo variants', 'de novo mutations identified in deCODE''s whole-genome sequencing of 15,220 Icelanders', 'Germline', 'Control Set', NULL, 'deCODE Genetics', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB19387', 'Archive of Transcriptomic Variants within the Sheep Milk Somatic Cells', 'Archive with all the variants detected within the sheep milk somatic cell transcriptome. Transcriptome sequencing (RNA-seq) was performed on total RNA extracted from milk somatic cells from ewes on days 10, 50, 120 and 150 after lambing. The experiment was performed in 4 Spanish Churra and 4 Assaf sheep, which differ in their milk production traits. The project Coordinator is Juan JosÃ© Arranz from University of LeÃ³n, Spain.', 'Germline', 'Control Set', NULL, 'University of LeÃ³n', 9940);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB15642', 'Molecular characterisation of Djallonke and Sahelian sheep breed genomes', 'We undertook the first whole genome variant analysis of the Djallonke and the Sahelian sheep breeds of Ghana. WGS from five individuals each of Djallonke and Sahelian sheep was sequenced on an Ion Proton. A total of approximately 404 million (97%) and 343 million (97%) sequence reads from the Djallonke and Sahelian genomes respectively were successfully mapped to the sheep reference genome  Oar_3.1/oviAri3. The mapped samples for each breed were then pooled. Variants were called/genotyped using a pipeline modified from GATK best practices.', 'Germline', 'Control Set', NULL, 'Curtin University', 9940);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB11749', 'HipSci imputed genotyping array data for healthy volunteers', 'The HipSci project brings together diverse constituents in genomics, proteomics, cell biology and clinical genetics to create a UK national iPS cell resource and use it to carry out cellular genetic studies. In this sub-study we performed imputed Genotyping analysis using the Infinium HumanExome BeadChip on iPS cells generated from skin biopsies from healthy volunteers.', 'Germline', 'Control Set', NULL, 'Wellcome Trust Sanger Institute', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB11750', 'HipSci genotyping array data for healthy volunteers', 'The HipSci project brings together diverse constituents in genomics, proteomics, cell biology and clinical genetics to create a UK national iPS cell resource and use it to carry out cellular genetic studies. In this sub-study we performed Genotyping analysis using the Infinium HumanExome BeadChip on iPS cells generated from skin biopsies from healthy volunteers.', 'Germline', 'Control Set', NULL, 'Wellcome Trust Sanger Institute', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB22989', 'Effect annotated whole genome variants for 163 vervet monkeys (SNPs, Indels, and macaque fixed differences)', 'Callset of filtered SNP and indel genotypes of 163 vervet monkeys (genus Chlorocebus) from several African and Caribbean countries as described in Svardal et al. Nature Genetics 2017. Sample information can be found in Data S1 of the publication above. There is one VCF file per chromosome. Genotypes are not imputed and unphased. There is missing data. Note that there also is a imputed and phased dataset of biallelic SNPs available as a separate submission which might be more useful for many applications. Fixed differences between vervet and macaque have been added to the VCFs (mainly the sites with AC=0 and AF=0.000) so that variant effects are also annotated for divergent sites between vervet and macaque. Where available, the macaque state is given by the INFO tag ''AA''. Variant effects have been annotated. Please refer to the methods section of the article cited above for a full description of the data. &lt;hannes.svardal@gmail.com&gt;', 'Germline', 'Control Set', NULL, 'Gregor Mendel Institute', 60711);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB17529', 'Personal Genome Project - United Kingdom', 'About PGP UK:Founded in 2013 by Stephan Beck, the United Kingdom Personal Genome Project is located at University College London.PGP UK is a member of the Global Network of Personal Genome Projects (PGP), a group of research studies creating freely available scientific resources that bring together genomic, environmental and human trait data donated by volunteers. Initiated by George Church at Harvard Medical School in 2005, the Personal Genome Project has pioneered ethical, legal, and technical aspects related to the creation of public resources involving highly identifiable data like human genomes.We believe sharing is good for science and society. Our project is dedicated to creating public resources that everyone can access. Privacy, confidentiality and anonymity are impossible to guarantee in a context like this research study where public sharing of genetic data is an explicit goal. Therefore, our project collaborates with participants who are fully aware of the implications and privacy concerns of making their data public. Volunteering is not for everyone, but the participants who join make a valuable and lasting contribution to science.We respect the people behind the data, and we aim to maintain strong relationships with participants. We want to collaborate on tracking health and other traits as they unfold over time. We also want to better understand the benefits and risks related to accessing and sharing personal genomes and other types of data.The genome is just a part of the story: genes interact with the environment to form traits. Participants may choose to contribute other public data to build public records of their health and traits. We also try to connect participants with research, education, and citizen science projects that are connected to personal genome data', 'Germline', 'Control Set', NULL, 'UNIVERSITY COLLEGE LONDON', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB9586', 'Deep genome sequencing for diverse human populations from around the world', 'The most powerful way to study population history and natural selection is to analyze whole genome sequences, which contain all the variation that exists in each individual. To date, genome-wide studies of history and selection have primarily analyzed data from single nucleotide polymorphism (SNP) arrays which are biased by the choice of which SNPs to include. Alternatively they have analyzed sequence data that have been generated as part of medical genetic studies from populations with large census sizes, and thus do not capture the full scope of human genetic variation. Here we supply high quality genome sequences (~40x average) from 301 individuals from 146 worldwide populations. All samples were sequenced using an identical protocol at the same facility (Illumina Ltd.). We modified standard pipelines to eliminate biases that might confound population genetic studies, to produce a unique alignment and genotype dataset.', 'Germline', 'Control Set', NULL, 'HARVARD MEDICAL SCHOOL, DEPARTMENT OF GENETICS', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB22189', 'Boar sperm DNA fragmentation', 'Variant detection in RNA-seq data from boars with high/low sperm DFI', 'Germline', 'Control Set', NULL, 'Norsvin SA', 9823);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB22833', 'Shull laboratory snps', 'Mammary cancer; congenic rat strain characterization.', 'Germline', 'Control Set', NULL, 'Rat Genome Database', 10116);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB21300', 'deCODE Release 2: whole-genome de novo variants from the proband from 1548 Icelandic trios', 'deCODE Release 2: whole-genome de novo variants from the proband from 1548 Icelandic trios', 'Germline', 'Control Set', NULL, 'deCODE genetics/Amgen Inc.', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB20726', 'Rotterdam Study Exome Sequencing Freeze 2', 'All variants detected by whole exome sequencing of 2628 Dutch healthy elderly individuals', 'Germline', 'Control Set', NULL, 'EMC', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB21794', 'OMIA LCV: Bulldog dwarfism in Dexter cattle', 'This project describes two likely causal variants (LCVs) for bulldog dwarfism in Dexter cattle', 'Germline', 'Control Set', NULL, 'OMIA', 9913);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB23068', 'RNA-Seq Analysis', 'RNA-Seq analysis carried out in three different backcrosses based on Iberian breed with Landrace, Pietrain and Duroc breeds', 'Germline', 'Control Set', NULL, 'INIA', 9823);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB22261', 'genetic variantion data for LYC', 'The SNP and indel (length less than 50bp) data from large yellow croaker (Larimichthys crocea)', 'Germline', 'Control Set', NULL, 'Key Laboratory of Healthy Mariculture for the East China Sea, Jimei University', 215358);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB22866', 'Public Science and Technology Research Funds Projects of Agriculture', 'To improve the technical standard and explore horse functional genome.', 'Germline', 'Control Set', NULL, 'Functional genome of Chinese native horse.', 9796);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB19794', 'The Danish reference pan genome phase II', 'Sequencing of Danish parent-offspring trios to determine genomic variation within the Danish population. Phase II comprises of 50 trios sequenced to 80X using Illumina with libraries of insert sizes from 180bp-20kb.', 'Germline', 'Control Set', NULL, 'Technical University of Denmark', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB20820', 'Deep whole-genome sequencing of 90 Han Chinese genomes', 'We performed whole genome sequencing at high depth (~80X) of 90 unrelated individuals of Chinese ancestry, collected from the 1000 Genome Project samples, including 45 North Han Chinese and 45 South Han Chinese samples. 83 of these 90 have been sequenced by the 1000 Genomes Project. We have identified 12,568,804 single nucleotide polymorphisms and 2,074,210 short InDels from these 90 samples.', 'Germline', 'Control Set', NULL, 'BGI-Shenzhen', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB20817', 'Haplotype-based stratification of Huntington''s disease', 'Genotypes of 21 common SNPs in Huntington''s disease subjects', 'Germline', 'Control Set', NULL, 'Center for Genomic Medicine, Massachusetts General Hospital', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB22799', 'GRCz11 Alternate Haplotype Clone Variants', 'The process of producing the GRC zebrafish assembly included the high-quality sequencing and finishing of clones representing alternate haplotypes to corresponding regions in the current primary assembly. This project reports the variation between those alternate haplotype clones and the primary assembly.', 'Germline', 'Control Set', NULL, 'Wellcome Trust Sanger Institute', 7955);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB22988', 'Imputed phased whole genome SNP variants for 163 vervet monkeys', 'Callset of filtered imputed and phased biallelic SNP genotypes of 163 vervet monkeys (genus Chlorocebus) from several African and Caribbean countries as described in Svardal et al. Nature Genetics 2017. Sample information can be found in Data S1 of Svardal et al. 2017.', 'Germline', 'Control Set', NULL, 'Gregor Mendel Institute', 60711);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB9799', 'Whole genome detection of sequences of six horses from diverse breeds.', 'Completed in 2009, the reference genome assembly of the domesticated horse (EquCab 2.0) produced the majority of publically available annotations of genetic variations in this species.  Following that effort, a few other projects that focused at variant discovery of a particular breed or two. In this project we aim to identify and annotate single nucleotide polymorphisms (SNPs), insertions and deletions (INDELs), copy number variations (CNVs) and structural variations (SVs) in the genomes six horses of diverse genetic background using next generation sequencing.', 'Germline', 'Control Set', NULL, 'UNIVERSITY OF FLORIDA', 9796);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB11746', 'HipSci whole exome sequencing data for healthy volunteers', 'The HipSci project brings together diverse constituents in genomics, proteomics, cell biology and clinical genetics to create a UK national iPS cell resource and use it to carry out cellular genetic studies.  In this sub-study we performed whole exome sequencing using Agilent whole exome pulldown method on iPS cells generated from skin biopsies from healthy volunteers.', 'Germline', 'Control Set', NULL, 'The Wellcome Trust Sanger Institute', 9606);
INSERT INTO projects(project_id, title, description, source_type, study_type, eva_center_name, center_name, taxonomy_id) VALUES('PRJEB11745', 'HipSci imputed whole exome sequencing data for healthy volunteers', 'The HipSci project brings together diverse constituents in genomics, proteomics, cell biology and clinical genetics to create a UK national iPS cell resource and use it to carry out cellular genetic studies.  In this sub-study we performed imputed whole exome sequencing using Agilent whole exome pulldown method on iPS cells generated from skin biopsies from healthy volunteers.', 'Germline', 'Control Set', NULL, 'The Wellcome Trust Sanger Institute', 9606);
